draft-ietf-ace-oscore-profile.xml

```diff
@@ -505,7 +505,7 @@ A5                                      # map(5)
         <section anchor="oscore-sec-ctx" title="OSCORE_Security_Context Object">
 
           <t>
-            An OSCORE_Security_Context is an object that represents part or all of an OSCORE Security Context, i.e., the local set of information elements necessary to carry out the cryptographic operations in OSCORE (Section 3.1 of <xref target="RFC8613"/>). In particular, the OSCORE_Security_Context object is defined to be serialized and transported between nodes, as specified by this document, but can also be used in this way by other specifications if needed. The OSCORE_Security_Context object can either be encoded as a JSON object or as a CBOR map. The set of common parameters that can appear in an OSCORE_Security_Context object can be found in the IANA "OSCORE Security Context Parameters" registry (<xref target="sec-ctx-params-reg"/>), defined for extensibility, and is specified below.
+            An OSCORE_Security_Context is an object that represents part or all of an OSCORE Security Context, i.e., the local set of information elements necessary to carry out the cryptographic operations in OSCORE (Section 3.1 of <xref target="RFC8613"/>). In particular, the OSCORE_Security_Context object is defined to be serialized and transported between nodes, as specified by this document, but can also be used as a representation of Security Context state by other specifications if needed. The OSCORE_Security_Context object can either be encoded as a JSON object or as a CBOR map. The set of common parameters that can appear in an OSCORE_Security_Context object can be found in the IANA "OSCORE Security Context Parameters" registry (<xref target="sec-ctx-params-reg"/>), defined for extensibility, and is specified below.
```

Comment:
I see the editorial improvement, but I have a problem with calling it "a representation of Security Context state", as the OSCORE_Security_Context does not contain all the informations of a full Security Context, for example no informations about Replay Window or sequence numbers, or derived keys... what about:

OLD:
but can also be used in this way by other specifications if needed.

NEW:
but can also be used by other specifications if needed.

Reply:
That should work, thanks.  The "in this way" is what was bothering me, as you surmised :)

Reply:
Do you want me to update the PR or can you handle things from your end?

Reply:
I can do it, thanks! :)

Reply:
Done in 6df6563.